core/templates/dev/head/player/PlayerServices.js

```diff
@@ -692,3 +748,53 @@ oppia.directive('feedbackPopup', ['oppiaPlayerService', function(oppiaPlayerServ
     }]
   };
 }]);
+
+oppia.controller('InformationCard', function ($scope, $modal) {
+
+  $scope.showInformationCard = function () {
+
+    var modalInstance = $modal.open({
+      animation: true,
+      templateUrl: 'popover/informationCard',
+      windowClass: 'oppia-modal-information-card',
+      controller: ['$scope', '$http', '$modal', '$modalInstance','oppiaPlayerService', 'ratingService',
+                   'oppiaHtmlEscaper', 'embedExplorationButtonService',
+                    function (
+                      $scope, $http, $modal, $modalInstance, oppiaPlayerService, ratingService, oppiaHtmlEscaper,
+                      embedExplorationButtonService) {
+
+        var _informationCardData = oppiaPlayerService.getInformationCardData();
+        $scope.serverName = window.location.protocol + '//' + window.location.host;
+        $scope.escapedTwitterText = oppiaHtmlEscaper.unescapedStrToEscapedStr(
+          GLOBALS.SHARING_OPTIONS_TWITTER_TEXT);
+        $scope.showEmbedExplorationModal = embedExplorationButtonService.showModal;
+
+        $scope.explorationId = oppiaPlayerService.getExplorationId();
```

Comment:
I think we need to add a TODO to turn both getExplorationId() and getExplorationTitle() into promises. The reason is, I loaded the page and immediately clicked on the info icon. This gave a console error since the title wasn't fetched yet. We probably should refactor so that these methods return promises instead.

It's fine to do this in a separate pass, though -- it'll require changes in other places and I don't think we should make the scope of this PR any larger. Hence the suggestion to add a TODO.